### PR TITLE
Update karafka and waterdrop. Added Interchanger controller

### PR DESCRIPTION
**Pull request by agwozdowski**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 # App class
 class App < Karafka::App
+  Dir[App.root + 'lib/**/*.rb'].each { |file| require file }
```
Comment (mensfeld) [inline app.rb]: this should be loaded automatically with loader

**Comment by agwozdowski** [comment]
@mensfeld example app updated with new waterdrop and karafka

File: app/controllers/aspected_messages_controller.rb [hunk 2]
```diff
@@ -13,7 +13,11 @@ class AspectedMessagesController < Karafka::BaseController
   # In this example it logs received params in 'log/aspect_controller_params.log' file
   def perform
     sleep 10
-    LoggerService.new.write_to_file(self, "#{Karafka::App.root}/log/aspect_controller_params.log")
+    LoggerService.new.write_to_file(
+      self,
+      "#{Karafka::App.root}/log/aspect_controller_params.log",
```
Comment (mensfeld) [inline app/controllers/aspected_messages_controller.rb]: Please use File.join()

File: app/controllers/basic_messages_controller.rb [hunk 3]
```diff
@@ -9,6 +9,10 @@ class BasicMessagesController < Karafka::BaseController
   # In this example it logs received params in 'log/basic_controller_params' file
   def perform
     sleep 10
-    LoggerService.new.write_to_file(self, "#{Karafka::App.root}/log/basic_controller_params.log")
+    LoggerService.new.write_to_file(
+      self,
+      "#{Karafka::App.root}/log/basic_controller_params.log",
```
Comment (mensfeld) [inline app/controllers/basic_messages_controller.rb]: File.join()

File: app/controllers/interchanger_messages_controller.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+require 'karafka'
```
Comment (mensfeld) [inline app/controllers/interchanger_messages_controller.rb]: do we need that?

**Comment by agwozdowski** [comment]
@mensfeld Removed ```require 'karafka'``` from controller and using ```File.join``` for log files

**Pull request merged**